lib/caesar.rb

```diff
@@ -1,6 +1,44 @@
 class Caesar
-  attr_reader :key
-
+  attr_reader :character_hash
   def initialize
+    @characters = ("a".."z")
```

Comment:
You were so close to getting case sensitivity! You would have just needed to do the same thing with the hash with the upper case characters.